capa/features/extractors/viv/insn.py

```diff
@@ -488,7 +488,7 @@ def extract_insn_segment_access_features(f, bb, insn):
 
 def get_section(vw, va):
     for start, length, _, __ in vw.getMemoryMaps():
-        if start <= va < start + length:
+        if va and start <= va < start + length:
```

Comment:
we should move this check out of the loop and raise an exception (`ValueError`?) if it isn't provided.

im afraid this might hide some other bug.

Reply:
```suggestion
        if start <= va < start + length:
```

Reply:
This code may be already hiding a bug, as your proposal makes that capa doesn't work anymore (at least with a few samples I tried it out) with both Python2 and Python3. `get_section` is called with `None` as some of the `va` returned by `insn.getBranches()` are `None` in `extract_insn_cross_section_cflow`. What about opening an issue about this? It is not really related to the Python 3 update.

Reply:
nice! yeah lets fix those bugs where they exist.

Reply:
#441 #442